actions/verify-hello-app/action.yml

```diff
@@ -0,0 +1,68 @@
+name: 'Verify Hello App'
+description: 'Ensure a hello-app style script outputs the expected response'
+inputs:
+  command:
+    description: 'Command to run the hello-app'
+    required: true
+  use_server_key:
+    description: 'Populate LAUNCHDARKLY_SERVER_KEY with a server-side SDK if true'
+    required: false
+    default: 'false'
+  use_client_key:
+    description: 'Populate LAUNCHDARKLY_CLIENT_KEY with a server-side SDK if true'
+    required: false
+    default: 'false'
+  use_mobile_key:
+    description: 'Populate LAUNCHDARKLY_MOBILE_KEY with a server-side SDK if true'
+    required: false
+    default: 'false'
+
+
+runs:
+  using: composite
+
+  steps:
+    - uses: launchdarkly/gh-actions/actions/release-secrets
+      if: ${{ inputs.use_server_key == 'true' }}
+      name: 'Get the server SDK key'
+      with:
+        aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+        ssm_parameter_pairs: '/sdk/common/hello-apps/server-key = LAUNCHDARKLY_SERVER_KEY'
+
+    - uses: launchdarkly/gh-actions/actions/release-secrets
+      if: ${{ inputs.use_client_key == 'true' }}
+      name: 'Get the client SDK key'
+      with:
+        aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+        ssm_parameter_pairs: '/sdk/common/hello-apps/client-key = LAUNCHDARKLY_CLIENT_KEY'
+
+    - uses: launchdarkly/gh-actions/actions/release-secrets
+      if: ${{ inputs.use_mobile_key == 'true' }}
+      name: 'Get the mobile SDK key'
+      with:
+        aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+        ssm_parameter_pairs: '/sdk/common/hello-apps/mobile-key = LAUNCHDARKLY_MOBILE_KEY'
+
+    - uses: launchdarkly/gh-actions/actions/release-secrets
+      name: 'Get the test feature flag key'
+      with:
+        aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+        ssm_parameter_pairs: '/sdk/common/hello-apps/boolean-flag-key = LAUNCHDARKLY_FLAG_KEY'
+
+    - name: 'Run hello-app command'
```

Comment:
Right now this is doing the basic testing we are doing in every other hello repo. But this provides us a natural extension point.

Instead of verifying a single boolean variable, we could increase the coverage for different flag types in the future. Or whatever type of expansion we might want to check.